src/Data/Graph/Typed.hs

```diff
@@ -1,3 +1,5 @@
+{-# language LambdaCase, ViewPatterns #-}
```

Comment:
Changes in this file seem accidental, please undo.

Reply:
Sorry, from my experiments running bfs on KGL representation over STUArray s Int Bools. The overhead from converting made this slower than working over AdjacencyIntMaps and IntSets, so I removed the code.